app/jobs/reports/verification_failures_report.rb

```diff
@@ -42,26 +42,34 @@ def verification_errors_data_for_issuers(date, report_name, issuers)
         end
       end
       data = csv.string
-      save_report("#{REPORT_NAME}.#{report_name}", data, extension: 'csv')
+      save_report("#{REPORT_NAME}/#{report_name}", data, extension: 'csv')
       data
     end
 
     def ial2_error_code(row)
       welcome_at = row['welcome_view_at']
       verify_at = row['verify_submit_at']
       phone_submit_at = row['verify_phone_submit_at']
-      doc_at = row['document_capture_submit_at']
       encrypt_at = row['encrypt_view_at']
       ssn_at = row['ssn_view_at']
       phone_view_at = row['verify_phone_view_at']
       return 'PHONE_FAIL' if submit_failed?(welcome_at, phone_submit_at, encrypt_at)
       return 'VERIFY_FAIL' if submit_failed?(welcome_at, verify_at, phone_view_at)
-      return 'DOCUMENT_FAIL' if submit_failed?(welcome_at, doc_at, ssn_at)
+      if submit_failed?(welcome_at, row['document_capture_submit_at'], ssn_at) ||
+         submit_failed?(welcome_at, row['back_image_submit_at'], ssn_at) ||
+         submit_failed?(welcome_at, row['capture_mobile_back_image_submit_at'], ssn_at) ||
+         submit_failed?(welcome_at, row['mobile_back_image_submit_at'], ssn_at)
+        return 'DOCUMENT_FAIL'
+      end
       'ABANDON'
     end
 
     def submit_failed?(welcome_at, submit_at, next_step_at)
-      submit_at && submit_at >= welcome_at && (!next_step_at || next_step_at < submit_at)
+      allow_recent_by_hours = 18 # this is tied to job frequency which is currently 24 hours
```

Comment:
yeah it's kinda arbitrary.  i guess i could make it 24 hours but didn't want it to repeat prior day's failures.

Reply:
I feel like 24 makes more sense than 18....I would use that if we can? Maybe we could try to read this from the config

```suggestion
      allow_recent_by_hours = 18 # this is tied to job frequency which is currently 24 hours
      cron_schedule = Rails.application.config.good_job.cron.find { |c| c[:class] == self.class.name }&.[](:cron)
      interval = cron_schedule ? Fugit.parse(cron_schedule).rough_frequncy.seconds : 24.hours
```
(in some environments, ex test or rails console, we don't load the cron schedule, so that's why we need to make it nil-safe)

Reply:
ok now that was cool!

Reply:
extreme like!

Reply:
problem is there is no guarantee it runs at exactly 24 hours after so we need a little wiggle room and definitely don't want to repeat prior day's failures

Reply:
ok so maybe we do like `(24 - 1).hours`? 18 is such a strange number for this, that's why it confused me

Reply:
sounds good

Reply:
`TypeError:  no implicit conversion of Symbol into Integer` digging further

Reply:
`cron_schedule = Rails.application.config.good_job.cron.values.find { |c| c[:class] == self.class.name }&.[](:cron)`